staging/src/k8s.io/kubectl/pkg/cmd/util/factory_client_access.go

```diff
@@ -159,19 +162,31 @@ func (f *factoryImpl) Validator(validate bool) (validation.Schema, error) {
 	}, nil
 }
 
-// OpenAPISchema returns metadata and structural information about Kubernetes object definitions.
+// OpenAPISchema returns metadata and structural information about
+// Kubernetes object definitions.
 func (f *factoryImpl) OpenAPISchema() (openapi.Resources, error) {
 	discovery, err := f.clientGetter.ToDiscoveryClient()
 	if err != nil {
 		return nil, err
 	}
-
-	// Lazily initialize the OpenAPIGetter once
+	// Lazily initialize the OpenAPI Resources once
 	f.openAPIGetter.once.Do(func() {
-		// Create the caching OpenAPIGetter
-		f.openAPIGetter.getter = openapi.NewOpenAPIGetter(discovery)
+		f.openAPIGetter.openAPIClient = openapi.NewOpenAPIGetter(discovery)
+		oapi, err := f.openAPIGetter.openAPIClient.OpenAPISchema()
+		if err != nil {
+			f.openAPIGetter.err = err
+			return
+		}
+		f.openAPIGetter.resources, f.openAPIGetter.err = openapi.NewOpenAPIData(oapi)
 	})
 
-	// Delegate to the OpenAPIGetter
-	return f.openAPIGetter.getter.Get()
+	return f.openAPIGetter.resources, f.openAPIGetter.err
+}
+
+func (f *factoryImpl) OpenAPIGetter() (discovery.OpenAPISchemaInterface, error) {
```

Comment:
Where is this called?

Reply:
Nowhere, but I was planning on using this for the dry-run verifier. Maybe not a good idea... I'll take a look around

Reply:
This is now used